docs/manual/3rd-party.md

```diff
@@ -0,0 +1,14 @@
+# Third-Party Tools and Extensions {#ch-3rd-party}
```

Comment:
I don't see this chapter show up in the built manual?

Reply:
It looks like I forgot the include in the top-level manual.md file

https://github.com/nix-community/home-manager/blob/2014be5ea9bc7ef0f934cd88ef45cfe4aa8831ae/docs/manual/manual.md?plain=1